diff --git a/node_modules/react-native-reanimated/ios/REAModule.mm b/node_modules/react-native-reanimated/ios/REAModule.mm
index a15b02d..d775bf8 100644
--- a/node_modules/react-native-reanimated/ios/REAModule.mm
+++ b/node_modules/react-native-reanimated/ios/REAModule.mm
@@ -115,12 +115,13 @@ - (void)installReanimatedUIManagerBindingAfterReload
 
   // to avoid deadlock we can't use Executor from React Native
   // but we can create own and use it because initialization is already synchronized
-  react_native_assert(self.bridge != nil);
-  RCTRuntimeExecutorFromBridge(self.bridge)(^(jsi::Runtime &runtime) {
-    if (__typeof__(self) strongSelf = weakSelf) {
-      [strongSelf injectDependencies:runtime];
-    }
-  });
+  if (self.bridge != nil) {
+    RCTRuntimeExecutorFromBridge(self.bridge)(^(jsi::Runtime &runtime) {
+      if (__typeof__(self) strongSelf = weakSelf) {
+        [strongSelf injectDependencies:runtime];
+      }
+    });
+  }
 }
 
 - (void)handleJavaScriptDidLoadNotification:(NSNotification *)notification
