diff --git a/node_modules/react-native-svg/android/build.gradle b/node_modules/react-native-svg/android/build.gradle
index e7b0d7a..c9d66c1 100644
--- a/node_modules/react-native-svg/android/build.gradle
+++ b/node_modules/react-native-svg/android/build.gradle
@@ -84,9 +84,5 @@ repositories {
 }
 
 dependencies {
-    if (isNewArchitectureEnabled()) {
-        implementation project(":ReactAndroid")
-    } else {
-        implementation 'com.facebook.react:react-native:+'
-    }
+    implementation 'com.facebook.react:react-native:+'
 }
diff --git a/node_modules/react-native-svg/apple/Elements/RNSVGImage.mm b/node_modules/react-native-svg/apple/Elements/RNSVGImage.mm
index 5a3cb91..7a8d615 100644
--- a/node_modules/react-native-svg/apple/Elements/RNSVGImage.mm
+++ b/node_modules/react-native-svg/apple/Elements/RNSVGImage.mm
@@ -37,6 +37,8 @@
 #import <rnsvg/RNSVGImageComponentDescriptor.h>
 #import "RNSVGFabricConversions.h"
 
+using namespace facebook::react;
+
 #endif // RN_FABRIC_ENABLED
 
 @implementation RNSVGImage {
@@ -50,7 +52,6 @@ @implementation RNSVGImage {
 #endif // RN_FABRIC_ENABLED
 }
 #ifdef RN_FABRIC_ENABLED
-using namespace facebook::react;
 
 - (instancetype)initWithFrame:(CGRect)frame
 {